test/format/en_US.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE formattest SYSTEM "../formattest.xsd">
+<formattest locale="en-US">
+    <test>
+        <params>
+            <param name="relationship" type="String"><value>brother</value></param>
+        </params>
+        <source>I didn't find <var name="relationship" inflect="indefinite"/> in your contacts. What is your <var name="relationship" inflect="genitive"/> name?</source>
+        <print>I didn't find a brother in your contacts. What is your brotherʼs name?</print>
+    </test>
+    <test>
+        <params>
+            <param name="relationship" type="String"><value>aunt</value></param>
+        </params>
+        <source>I didn't find <var name="relationship" inflect="indefinite"/> in your contacts. What is your <var name="relationship" inflect="genitive"/> name?</source>
+        <print>I didn't find an aunt in your contacts. What is your auntʼs name?</print>
+    </test>
+    <test>
+        <params>
+            <param name="object" type="String"><value>light</value></param>
+        </params>
+        <source>The <var name="object"/> <switch value="object" feature="number">
+            <case is="plural">are</case>
+            <default>is</default>
+        </switch> on</source>
+        <print>The light is on</print>
+    </test>
+    <test>
+        <params>
+            <param name="object" type="String"><value>lights</value></param>
+        </params>
+        <source>The <var name="object"/> <switch value="object" feature="number">
```

Comment:
Is the idea here that the implementation is capable of inspecting the object (`lights`) and extracting some grammatical information from it? (here: the grammatical number).

Reply:
Yes